### DISCUSSION TITLE
"Show only errors" when importing CBL/Reading list

### DISCUSSION THREAD

**Discussion by rad96** [gotson/komga#1702] [Ideas]
When importing large CBLs it's hard to find problematic issues, which are stopping you from actually creating Reading list. It would be nice to have some kind of checkbox to filter out all issues and show only ones that have some kind of problem - duplicates or not found.

**Comment by gotson** [comment]
I've added filters for OK, error, and duplicate:
<img width="1196" alt="image" src="https://github.com/user-attachments/assets/31cf149b-84dd-4c77-9d14-f205e600e301" />

**Comment by gotson** [comment]
🎉 This issue has been resolved in `1.19.0` ([Release Notes](https://github.com/gotson/komga/releases/tag/1.19.0))
